examples/spring-reactive/README.md

```diff
@@ -0,0 +1,25 @@
+# Spring Reactive + CloudEvents sample
+
+## Build
+
+```shell
+mvn package
+```
+
+## Start HTTP Server
+
+```shell
+mvn spring-boot:run
+```
+
+You can try sending a request using curl, and it echos back a `Foo` with new `ce-*` headers:
+
+```shell
+curl -v -d '{"value": "Foo"}' \
+    -H'Content-type: application/json' \
+    -H'ce-id: 1' \
+    -H'ce-source: cloud-event-example' \
+    -H'ce-type: my.application.Foo' \
+    -H'ce-specversion: 1.0' \
+    http://localhost:8080/foos
```

Comment:
I think we need a little bit more docs here:

* Add 2 curl for the `/event` endpoint, one for binary and one for structured path, to show it accepts both binary and structured
* Be explicit here that `/foos` accepts only binary cloudevents
* Replace _and it echos back a `Foo` with new `ce-*` headers_ with _and it echos back a CloudEvent containing `Foo`

Reply:
OK. Note that the other examples have less docs though.